flickit-assessment-core/src/main/java/org/flickit/assessment/core/adapter/out/persistence/kit/attribute/AttributePersistenceJpaAdapter.java

```
@@ -128,4 +133,49 @@ public Attribute load(Long attributeId, Long kitVersionId) {
     public int countAttributes(long kitVersionId) {
         return repository.countByKitVersionId(kitVersionId);
     }
+
+    @Override
+    public List<LoadAttributesPort.Result> loadAttributes(UUID assessmentId) {
+        var views = repository.findAllAttributes(assessmentId);
+        var attributes = views.stream()
+            .map(e -> AttributeMapper.mapToDomainModel(e.getAttribute()))
+            .toList();
+
+        var assessmentResultEntity = assessmentResultRepository.findFirstByAssessment_IdOrderByLastModificationTimeDesc(assessmentId)
```

این کار بهتره که ابتدای متد انجام بشه و fail fast داشته باشیم.
کلا هم به نظرم بهتره به جای لود نتیجه ارزیابی، صرفا خود ارزیابی رو با findByIdAndDeletedFalse لود کنیم (باید متدش ایجاد بشه تو ریپازیتوریش).